programs/tensorflowDetection2D/MainDetector.cpp

```diff
@@ -0,0 +1,226 @@
+
+// Librerias
+
+#include <fstream>
+#include <utility>
+#include <vector>
+#include <iostream>
+#include <time.h>
+#include <yarp/os/all.h>
+#include <yarp/sig/all.h>
+#include <yarp/sig/Image.h>
+#include <yarp/os/Time.h>
+#include <opencv2/core/mat.hpp>
+#include <opencv2/videoio.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <cv.hpp>
+#include "tensorflow/cc/ops/const_op.h"
+#include "tensorflow/cc/ops/image_ops.h"
+#include "tensorflow/cc/ops/standard_ops.h"
+#include "tensorflow/core/framework/graph.pb.h"
+#include "tensorflow/core/graph/default_device.h"
+#include "tensorflow/core/graph/graph_def_builder.h"
+#include "tensorflow/core/lib/core/threadpool.h"
+#include "tensorflow/core/lib/io/path.h"
+#include "tensorflow/core/lib/strings/stringprintf.h"
+#include "tensorflow/core/platform/init_main.h"
+#include "tensorflow/core/public/session.h"
+#include "tensorflow/core/util/command_line_flags.h"
+#include "TensorflowDetector.hpp"
+#include "MainDetector.hpp"
```

Comment:
Are all those includes mandatory? Also, `<map>` is probably missing.

Reply:
Map library added at 5de346cdbea0202d1187e404122427aa29888006.